calico/etcdutils.py

```diff
@@ -135,7 +149,8 @@ class EtcdWatcher(EtcdClientOwner):
     def __init__(self,
                  etcd_authority,
                  key_to_poll):
-        super(EtcdWatcher, self).__init__(etcd_authority)
+        super(EtcdWatcher, self).__init__(etcd_authority, "http", None, None,
+                                          None)
```

Comment:
I'm not thrilled about this workaround, TBH... Was thinking about having a "EtcdHttpClientOwner" that used the basic __init__ for EtcdClientOwner or allowing the __init__ of EtcdClientOwner to have the keys and certs as keyword args that default to None.